README.md

```diff
@@ -42,6 +42,7 @@ each configuration option. They are all required.
 
     # Basic Mina requirements probably live here...
 
+    set :git_branch, ENV['branch'] || 'master' # Your specifc git branch to deploy
```

Comment:
Can we consider using simply `branch` here to match how Mina/Cap handle branches with their various SCM plugins?